### DISCUSSION TITLE
Interplanetary Community Ambassadors Pilot Program - Filecoin Track

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/community#116] [News and Events]
# Interplanetary Community Ambassadors Pilot Program - Filecoin Track 

🛰 We are excited to introduce the **Interplanetary Community Ambassadors Pilot Program - Filecoin Track** to you all!

Although compare to other Interplanetary communities like the IPLD and IPFS community, the Filecoin community is still young, however it is very thriving, active and incredibly supportive. Therefore, we are very excited to create a new opportunity for individual community members to continue to steward the Filecoin community as ambassadors.

This **6-week pilot program** will focus on enabling each community ambassador with the skills to support active engagement in the community, as well as develop crucial community management skills, and learn new ways to support Filecoin miners, clients and developers.

## ⁉️ **Why is this important?** 

Filecoin's mission is to create a decentralized, efficient, and robust foundation for humanity’s information. There are thousands of community members, developers, users, miners, and partners that want to contribute to that mission, but can't get started because of of a lack of support early on. The Community Ambassadors Program is crucial to the success of Filecoin - help onboard and steward thousands of new community members to further Filecoin's mission.    

## 🤔 **How does it work?**

- An Ambassador’s responsibilities:
    - Help onboard new miners, clients and developers to the network by directing them to the relevant resources available.
    - Help moderate discussions on Slingshot and Filecoin Plus.
    - Help with encouraging and engaging FIP discussions among the community.
    - Ambassadors who are miners will be helping with miner community technical support for Lotus.
    - Weekly sync within the initial cohort and Filecoin Ambassador lead.
    - Create weekly community reports detailing highs and lows, metrics, etc.
- Expected effort: 4 hours per week.
- Training will be provided.

## 🧪 **This is a pilot**

The initial Interplanetary Community Ambassador Pilot program’s scope and structure are WIP and iterative. To make the pilot program successful, the first cohort will be limited to a select group of 5-8 community members. 

## 🏆 **Community Ambassadors Rewards**

- Being showered with all the Filecoin swag you can handle
- Office hours with Protocol Labs/core devs
- Performance pending—opportunities to lead future cohorts
- **Additional incentives** to be announced in the future 😉

## 🙋‍♀️ **Interested? Here’s how to apply**

- Application closed.
- Selected ambassadors will be notified via Slack DM/email
- Stay tuned for future cohort opportunities!
- Questions? Leave a comment in this discussion.

**Comment by jennijuju** [comment]
We have received an overwhelming response to this program and thank you to those who have applied to participate in the program! We are currently reviewing the applications and will keep in touch with you soon! 💙

**Reply by XL202205** [reply]
Hi Jennifer,

我们想做存储投资，请问项目方在中国有联系人吗？能不能提供联系方式。

**Reply by a1017412510** [reply]
> 

联系我，加我V：laolinshuo
我这边是三峡星，专注服务B端客户，我们有全产业链的服务，IDC机房位于三峡数据中心那边。

**Comment by jennijuju** [comment]
## Thank you for applying

🛰 Thank you for all your interest in the Interplanetary Community Ambassadors Pilot Program - Filecoin Track! We were overwhelmed by the volume of responses we received, and could only include 8 participants for this first pilot. 

🎉 That being said, the first cohort of Interplanetary Community Ambassadors has been selected and we will share their profiles with the community in the following week! We are excited to have them for supporting Filecoin miners, clients, and developers. Thank you all for volunteering to join us on this mission to help onboard and steward thousands of new community members to further Filecoin's mission. 

🔖 To those community members that we had to pass this time sadly, please stay tuned, this pilot is just a small initial test. There will be more opportunities to join an upcoming Interplanetary Community Ambassadors program round soon.

**Reply by LEODI83** [reply]
Hello, Jennifer!

Can an ordinary person become an ambassador or how to join a project?)

If I don't use Slack
Where can I communicate and find information?
For example, are you planning to open TG?

**Reply by waynewyang** [reply]
Hi you can get infomation as belows,
## [filecoin docs](https://docs.filecoin.io/)
## [filecoin lotus discussions](https://github.com/filecoin-project/lotus/discussions)
## [filecoin lotus Issue](https://github.com/filecoin-project/lotus/issues)
## [filecoin github community](https://github.com/filecoin-project/community)
## [filecoin discussions](https://github.com/filecoin-project/community/discussions)
## [proto.school](https://proto.school/)
## [Blog](https://filecoin.io/zh-cn/blog/)
## [filecoin research](https://research.filecoin.io/)
## [filecoin SECURITY](https://github.com/filecoin-project/community/blob/master/SECURITY.md)
## [Youtube](https://www.youtube.com/channel/UCPyYmtJYQwxM-EUyRUTp5DA)
## [bilibili](https://space.bilibili.com/1644606620?spm_id_from=333.788.b_765f7570696e666f.2)
## [filecoin-spec](https://spec.filecoin.io/#section-intro)
## [running filecoin network](https://network.filecoin.io/)
## [filecoin-plus](https://filplus.info/#/)

**Reply by jennijuju** [reply]
> Hello, Jennifer!
> 
> Can an ordinary person become an ambassador or how to join a project?)
> 
> If I don't use Slack
> Where can I communicate and find information?
> For example, are you planning to open TG?

Of course, in the future, we would love to open up more opportunities for different roles on different platforms. However, Slack is our main platform for communication and most of our community is on it, so I'd suggest you join it soon, it's fun!

**Comment by jennijuju** [comment]
# Meet The Ambassadors!

🛰Meet the first cohort of Filecoin Ambassadors below, and find them on GitHub, Filecoin Slack(@fil-community-ambassadors) and etc if you have any questions!

**Comment by jennijuju** [comment]
# Hunter Trujillo 

Blockchain Developer at ChainSafe Systems

<img width="294" alt="image" src="https://user-images.githubusercontent.com/42981373/114919104-d91c7000-9df5-11eb-9834-a573f18a29a4.png">

I’m a blockchain developer at ChainSafe, and I work on the Rust implementation of Filecoin. I’ve worked in the tech industry for nearly 10 years, and I’ve been working with Rust and blockchain tech since 2017. I’m also an entrepreneur, consultant, cat dad, and loving husband. I look forward to doing my part so users can fight back against the cabal of rent-seeking cloud companies, and instead shop around within a user-friendly storage marketplace.

**Location:** Longmont, Colorado
**Slack:**  Hunter Trujillo [ChainSafe]
**Github:**  @cryptoquick
**Twitter:** [cryptoquick](https://twitter.com/cryptoquick)
 **LinkedIn:** https://www.linkedin.com/in/huntertrujillo/

## Spirit

<img width="307" alt="image" src="https://user-images.githubusercontent.com/42981373/114919395-30224500-9df6-11eb-93aa-314757207c98.png">
Spirit is an expert at the fine art of being cute. Likes: relaxing in warm places, receiving lots of attention. Dislikes: closed doors, being moved.


**Location:** Longmont, Colorado (MST)
**Slack:** Ping Hunter Trujillo if you need me

**Comment by jennijuju** [comment]
# Kimmy Wen 

Filecoin miner @ KeyPool, Founder @ Flame & Filbox

<img width="258" alt="image" src="https://user-images.githubusercontent.com/42981373/114926142-ffdea480-9dfd-11eb-8195-2dce109b3cf2.png">

I’m dedicated to harnessing the Filecoin mining community and ecosystem building, so if you have any questions or an idea you want to discuss with, feel free to reach out to me on slack, wechat or LinkedIn - happy to have a chat. 

**Location:**   Shanghai, China
**Slack:**   kimmy.w
**Wechat:**  BazingaS
 **LinkedIn:**  Kimmy Sijie Wen

**Comment by jennijuju** [comment]
# Max Yu 

Developer of CoinSummer

<img width="223" alt="image" src="https://user-images.githubusercontent.com/42981373/114926335-3a484180-9dfe-11eb-81c8-61518064a4c8.png">

Co-founder of CoinSummer Lab. I have been a Web Developer and Product Manager for about 6 years, and worked on image recognition use deep learning two years ago. I am focused on Filecoin and Polkadot development and research currently. 

**Location:** Beijing, China
**Slack:**  Max
**Github:**  @yuwenhui
**Wechat:** max-yu-666

**Comment by jennijuju** [comment]
# Ørjan R

<img width="342" alt="image" src="https://user-images.githubusercontent.com/42981373/114926518-711e5780-9dfe-11eb-876b-a62a50bfca0c.png">

I have been interested in both Filecoin and IPFS since I read the Filecoin v1 whitepaper back in 2015. I currently operate a small miner in Oslo, Norway, and look forward to support the amazing Filecoin-community. When I’m not available on Slack you can probably find me hiking in the mountains, or snowboarding.

**Location:**  Oslo, Norway
**Slack:**   Phi-rjan
**Github:**   @rjan90
**Wechat:** Phi-rjan
**Twitter:** [orjanroren](twitter.com/orjanroren)

**Comment by jennijuju** [comment]
# Patrick 

Fluxbox user

<img width="493" alt="image" src="https://user-images.githubusercontent.com/42981373/114928282-69f84900-9e00-11eb-9c46-5161b7c84e57.png">

If it looks good in the end - fine, but the primary objective is to make it work.  

**Location:**  Berlin, Germany
**Slack:**  Patrick - Factor8 Solutions
**Github:**  @f8-ptrk
**Wechat:** Factor8-Solutions

**Comment by jennijuju** [comment]
# James B 

Filecoin Miner

<img width="508" alt="image" src="https://user-images.githubusercontent.com/42981373/114929959-78476480-9e02-11eb-87e4-955d9197c7ac.png">

I have been a part of the blockchain space since 2015 and joined the Filecoin community as a small miner in August of 2020, based in the (sometimes) sunny UK. Please feel very welcome to reach out at any time, I am always happy to help!! 

**Location:**  United Kingdom, Oxford 
**Slack:**  TippyFlits
**Github:**  @TippyFlitsUK
**Twitter:** TippyFlits

**Comment by jennijuju** [comment]
# waynewyang

filecoin enthusiastic

<img width="598" alt="image" src="https://user-images.githubusercontent.com/42981373/114931570-68308480-9e04-11eb-8981-b4d896217a7d.png">

I agree with the mission of IPFS/Filecoin and hope to do some interesting development work in the IPFS/filecoin ecosystem.

**Location:**  Shenzhen,China
**Slack:**  waynewyang
**Github:**  @waynewyang
**Twitter:** [waynewyang](https://twitter.com/waynewyang
) **LinkedIn:** https://www.linkedin.com/in/waynewyang/

**Comment by jennijuju** [comment]
# Charles Cao 

Founder of Nebula AI

<img width="299" alt="image" src="https://user-images.githubusercontent.com/42981373/114918469-1af8e680-9df5-11eb-803f-6fe692753d5e.png">

I am dedicated to blockchain technology since 2017,.Decentralized cloud computing and ledger technology is the most fantastic thing in the world. My personal hobbies include researching new technologies which can change the way people live and activities help people make their life more positive.

**Location:**  Montreal Canada
**Slack:**  Charles Cao -filswan
**Github:**  @flyworker
**Wechat:** flywind08
**Twitter:** [flywind08](https://twitter.com/flywind08)
**LinkedIn:** https://www.linkedin.com/in/charles-cao-09a79526

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
please see:
https://filecoin.notion.site/Welcome-to-the-Interstellar-Ambassador-Program-40377d2b7f33457593528cb707609e13
https://orbitcommunity.filecoin.io/

for the live ambassador programs. the pilot is being replaced by them.
